src/Admin/AdherentAdmin.php

```diff
@@ -384,6 +397,14 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
         ;
     }
 
+    /**
+     * @param Adherent $adherent
+     */
+    public function postUpdate($adherent)
```

Comment:
method signature should not change

Reply:
I read about this yesterday and PHP does not enforce the name of parameters. It only enforces the number of **required** parameter and their types. Thus, you can add any optional argument as you want and also add return type if the parent method does not specify one.

Anyway I'll change it as you proposed because I am not a big fan either :+1: